src/cljs/main/broadfcui/main.cljs

```diff
@@ -209,6 +191,11 @@
                                public?
                                (contains? (:user-status @state) :signed-in))]
        [:div {}
+        (let [error (:force-sign-in-error @state)]
+          (when error
+            (modals/render-error {:header (str "Error validating access token")
```

Comment:
Could be combined as: `(when-let [error (:force-sign-in-error @state)] ...)`

Reply:
More learning. Thanks!